app/models/ingredient.py

```diff
@@ -4,5 +4,10 @@
 
 class Ingredient(BaseModel):
     id: UUID4
+    amount: float  # maybe make a way to scale the recipe by multiplying the amount variable?
+    measurementType: str  # it should be possible to leave this blank, i.e. 1 ___ lemon vs 1 tsp cumin

```

Comment:
Python doesn't use camel or pascal case often, the preferred/idiomatic/pythonic way of naming variables would be like "meaurement_type".

Reply:
implemented